src/cli_keystore/add.test.js

```diff
@@ -9,8 +9,6 @@ import * as prompt from '../server/utils/prompt';
 
 describe('Kibana keystore', () => {
   describe('add', () => {
-    const sandbox = sinon.sandbox.create();
```

Comment:
__note:__ `sinon` is now a (default) sandbox, see http://sinonjs.org/guides/migrating-to-5.0